src/java/main/org/apache/zookeeper/server/quorum/LearnerHandler.java

```diff
@@ -792,7 +792,14 @@ public boolean syncFollower(long peerLastZxid, ZKDatabase db, Leader leader) {
                     txnProposalItr.close();
                 }
             } else {
-                LOG.warn("Unhandled scenario for peer sid: " +  getSid());
+                LOG.warn("Unhandled scenario for peer sid: {} maxCommittedLog=0x{}"
```

Comment:
The same context is also provided at line 703 except the txnLogSyncEnabled, looks like nothing changed between these lines, any reason we added this extra logging here?

Reply:
the logging levels are different, and it is nice to have the evaluation information with weird corner case scenario. i think it's worth surfacing the information here as well.